crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -938,6 +942,31 @@ mod private {
             &self.events
         }
 
+        /// Find a single event in this room.
+        pub async fn event(
+            &self,
+            event_id: &EventId,
+        ) -> Result<Option<(Position, TimelineEvent)>, EventCacheError> {
```

Comment:
What's the value of returning the position? No callers make uses of it. (If the answer is "but maybe later they do!" my counter-answer is "then let's add it later :nerd_face:")

Reply:
We need it for `maybe_apply_new_redaction` 😛.

Reply:
I hear the song of "we need it later" 😏

Reply:
hehe, but this PR is a prequel of the work for `maybe_apply_new_redaction`, I should have shared that.